service/history/replication/task_processor_test.go

```diff
@@ -167,6 +172,167 @@ func (s *taskProcessorSuite) TestSendFetchMessageRequest() {
 	s.NotNil(requestMessage.respChan)
 }
 
+func (s *taskProcessorSuite) TestProcessorLoop_RespChanClosed() {
+	// start the process loop
+	s.taskProcessor.wg.Add(1)
+	go s.taskProcessor.processorLoop()
+	defer close(s.taskProcessor.done)
+
+	// act like taskFetcher here and populate respChan of the request
+	requestMessage := <-s.requestChan
+	close(requestMessage.respChan)
+	time.Sleep(50 * time.Millisecond)
```

Comment:
since we have time.After(50 * time.Millisecond), can we remove this time.Sleep?

Reply:
makes sense. this is not needed.